lib/matplotlib/font_manager.py

```diff
@@ -1099,15 +1099,16 @@ def score_family(self, families, family2):
         Returns a match score between the list of font families in
         *families* and the font family name *family2*.
 
-        An exact match anywhere in the list returns 0.0.
+        An exact match at the head of the list returns 0.0.
 
-        A match by generic font name will return 0.1.
+        A match further down the list will return between 0 and 1.
 
         No match will return 1.0.
         """
         if not isinstance(families, (list, tuple)):
             families = [families]
         family2 = family2.lower()
+        step = 1 / len(families)
```

Comment:
I think we need a special case here for `len(families)` == 0.  Unlikely, but technically possible.